.github/workflows/manifests.yaml

```diff
@@ -0,0 +1,32 @@
+name: manifests-diff
+
+on:
+  pull_request:
+  push:
+    branches:
+    - main
+
+jobs:
+  manifests-diff:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout target branch of repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.base_ref }}
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: "go.mod"
+      - name: Generate rukpak.yaml installation manifest from target branch
+        run: make quickstart
+      - name: Rename manifest file for later comparison
+        run: mv rukpak.yaml rukpak-TODO.yaml
+      - name: Checkout PR branch
+        uses: actions/checkout@v3
+        with:
+          clean: false # Preserve the manifest we generated earlier
+      - name: Generate rukpak.yaml installation manifest from source branch
+        run: make quickstart
+      - name: Compare generated manifests of source and target branch
+        run: diff rukpak.yaml rukpak-TODO.yaml
```

Comment:
I think `diff` will be unreliable as it compares line by line. So something like this will give a false positive:

```
key1: val1
key2: val2
```
vs
```
key2: val2
key1: val1
```

Files are different. But semantically - they are the same.

Reply:
This is a good point - I may need to put some more work in to refine the comparison.

Reply:
fwiw, `encoding/json` (which I think is ultimately what is used under the hood of kustomize to encode/decode) always sorts map keys lexicographically.

Not positive, but I don't think we'd see this particular scenario. But if something moves a list item around, that would result in a non-semantic-difference diff.